src/components/RegisterComponent/RegisterComponent.jsx

```diff
@@ -10,26 +10,13 @@ class RegisterComponent extends Component {
       name: "",
       email: "",
       password: "",
-      "confirm password": "",
-      height: 200,
-      weight: 75,
-      activity_level_id: 1,
-      birth_date: "1999-01-01",
-      gender_id: 1,
-      user_tier_id: 1,
-      goal_id: 1,
-      formErrors: {
-        name: "",
-        email: "",
-        password: "",
-        "confirm password": "",
-        height: "",
-        weight: "",
-        activity_level: "",
-        birth_date: ""
-      }
+      confirm_password: ""
     };
   }
+  
+  componentDidMount() {
+    this.props.dispatchGetEmails()
```

Comment:
I think this works for the demo, however, in the future it would be quite dangerous to have an array on the front end that shows all of the emails saved to our database and we would have to change this.

Reply:
I'll change it to filter through the backend.

Reply:
just updated the emails function. Let me know if you find anything, else I'll merge.